vars/piperPipelineStageRelease.groovy

```diff
@@ -95,6 +97,12 @@ void call(Map parameters = [:]) {
             }
         }
 
+        if(config.tmsExport){
+            durationMeasure(script: script, measurementName: 'export_release_tms_duration') {
+                tmsExport script: script
+            }
+        }
```

Comment:
It doesn't make a lot of sense to use tmsUpload and tmsExport at the same time.
Can we skip here tmsExport if tmsUpload  is configured?
Something like this:
```
if (config.tmsUpload) {
 ...
}else if(config.tmsExport){

}
```